ui/Screen/Org/Fund.svelte

```diff
@@ -13,9 +14,11 @@
   };
 </script>
 
-<Remote {store} let:data={transactions}>
-  <Wallet
-    transactions={orgTransactions(transactions)}
-    balance="3484"
-    address="hyda4fhdx8up8hhocagkdz3d41txb98stw4pkqe3uommo1p5m6s9oy" />
+<Remote store={txStore} let:data={transactions}>
```

Comment:
The big benefit of the `Remote` component is simplicity, which we lose if we nest them. Unless you're doing separate loading states for each of these entities, it makes more sense to use the reactive store for single values within `<script>`.

Reply:
Good point. https://github.com/radicle-dev/radicle-upstream/pull/691/commits/3899cd8c8fdda44ba868d618a78a9a20c485f9cc